File: inflearn/algorithm/1-01/main.js
const ps = require("../modules/ps");

const solution = (str, ch) => {
  let answer = 0;
  str = str.toLowerCase(); // 문자열을 소문자로 변환
  for (let i = 0; i < str.length; i++) {
    if (str[i] === ch) {
      answer++;
    }
  }
  return answer;
};

ps.main(async () => {
  let input1 = await ps.readLine(); // 문자열 입력
  let input2 = await ps.readLine(); // 문자열 입력

  console.log(solution(input1, input2));
});

File: inflearn/algorithm/1-01/main.dart
import 'dart:io';

int solution(String str, String ch) {
  int answer = 0;
  str = str.toLowerCase();
  for (var s in str.split('')) {
    if (s == ch) {
      answer++;
    }
  }
  return answer;
}

void main() {
  String input1 = stdin.readLineSync()!; // 문자열 입력
  String input2 = stdin.readLineSync()![0]; // 문자 입력

  print(solution(input1, input2));
}
